framework/test/utils/random.js

```diff
@@ -34,7 +34,9 @@ const random = {};
 
 // Returns a random number between min (inclusive) and max (exclusive)
 random.number = function(min, max) {
```

Comment:
convert to TS?

Reply:
ill do in different PR, it's already too big i think